make-containerized-build.groovy

```diff
@@ -27,12 +23,15 @@ node('cico-workspace') {
 
 	try {
 		stage('prepare bare-metal machine') {
-			sh 'scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ./prepare.sh root@${CICO_NODE}:'
-			sh 'ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@${CICO_NODE} ./prepare.sh --workdir=/opt/build --gitrepo=${GIT_REPO} --branch=${GIT_BRANCH}'
-		}
-
+			if ("${ghprbPullId}".length() != 0){
```

Comment:
Can you correct the indention? Use tabs everywhere, not a mix of tabs+spaces as below.

Reply:
Okay, I'll keep that in mind